index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Shop</title>
+</head>
+
+<body>
+    <header>
+        <section>
+            <a href="/index.html">Shop</a>
+            <input type="text">
+            <input type="button" value="Search">
+        </section>
+        <hr />
+        <section>
+            <nav>
+                <a href="#photo_video">Photo &amp; video</a> |
+                <a href="#change_the_way">Change the Way You Use Light</a> |
+                <a href="#product_information">Product information</a>
+            </nav>
+        </section>
+        <hr />
+    </header>
+    <section>
+        <h1>Philips Hue White and Color Ambiance A19 60W Equivalent LED Smart Bulb Starter Kit</h1>
+        <p> Price: 100 &euro;
+            <input type="button" value="Buy now">
+        </p>
+        <ul>
+            <li><a href="mailto:buy@shop.com">Order via email buy@shop.com</a></li>
+            <li><a href="tel:(111) 1111111">Order via call (111) 1111111</a></li>
+        </ul>
+    </section>
+    <main>
+        <section id="photo_video">
+            <h1>Photo &amp; Video</h1>
```

Comment:
```suggestion
            <h2>Photo &amp; Video</h2>
```

на странице может быть только один `<h1>`